test/parallel/test-fs-rmdir-recursive-throws-not-found.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+tmpdir.refresh();
+
+{
+  assert.throws(
+    () =>
+      fs.rmdirSync(path.join(tmpdir.path, 'noexist.txt'), { recursive: true }),
+    {
+      code: 'ENOENT',
+    }
+  );
+}
+{
+  fs.rmdir(
+    path.join(tmpdir.path, 'noexist.txt'),
+    { recursive: true },
+    common.mustCall((err) => {
+      assert.strictEqual(err.code, 'ENOENT');
+    })
+  );
+}
+{
```

Comment:
mind adding a comment above each block describing the test case?

Reply:
There all the same test case, one for each `rmdir` method. Do you have suggestions for comments that are not redundant?

```js
// Testing fs.rmDirSync.
// Testing fs.rmDir.
// Testing fs.promises.rmDir.
```

Reply:
I suppose you're right, the comments can be omitted.